src/feature-parameters.ts

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var limit = 2000;
-var files = -1;
+var files = 2;
```

Comment:
Czemu domyślna wartość została zmieniona? `-1` oznaczało, że górnym limitem staje się możliwa ilość wynikowych plików, a teraz domyślne to `2` - przez tę zmianę ignorowana jest cała logika licząca automatycznie wynikową liczbę plików.